src/partials/resource-limits.md

```diff
@@ -0,0 +1,62 @@
+## Check resource usage {% #check-resource-usage %}
+
+You can check your organization's resource usage by navigating to the overview page of your organization. 
+Under the **Usage** tab, you can check the current billing cycle's usage stats.
+
+
+## Reaching resource limits {% #reaching-resource-limits %}
+The following limit consequences will persist until the current billing period ends.
+{% table %}
+* Component
+* Consequence
+---
+* **Bandwidth** {% rowspan=3 %}
+*  More bandwidth will be purchased automatically until organization reaches budget cap. If the organization uses a Starter plan or budget cap is reached, API access will be disabled until more bandwidth is purchased.
```

Comment:
For free it makes sense you `until more bandwidth is purchased`, but for when you are pro already, solution is to increase budget cap.

Reply:
Fixed